src/test/java/seedu/address/logic/commands/DeleteModuleCommandTest.java

```diff
@@ -28,25 +29,29 @@
     public void executeValidCodeUnfilteredListSuccess() {
         DeleteModuleCommand deleteModuleCommand = new DeleteModuleCommand(DISCRETE_MATH.getCode());
 
-        String expectedMessage = String.format(DeleteModuleCommand.MESSAGE_DELETE_MODULE_SUCCESS, deleteModuleCommand);
+        String expectedMessage = String.format(DeleteModuleCommand.MESSAGE_DELETE_MODULE_SUCCESS,
+                deleteModuleCommand);
 
         ModelManager expectedModel = new ModelManager(model.getTranscript(), new UserPrefs());
         expectedModel.deleteModule(target -> target.getCode().equals(DISCRETE_MATH.getCode()));
         expectedModel.commitTranscript();
 
-        assertCommandSuccess(deleteModuleCommand, model, commandHistory, expectedMessage, expectedModel);
+        assertCommandSuccess(deleteModuleCommand, model, commandHistory, expectedMessage,
+                expectedModel);
     }
 
     @Test
     public void executeInvalidCodeUnfilteredListThrowsCommandException() {
-        Code invalidCode = new Code("CS9999");
+        Code invalidCode = new Code(TypicalModules.CODE_SOFTWARE_ENGINEERING);
         DeleteModuleCommand deleteModuleCommand = new DeleteModuleCommand(invalidCode);
 
-        assertCommandFailure(deleteModuleCommand, model, commandHistory, Messages.MESSAGE_INVALID_MODULE);
+        assertCommandFailure(deleteModuleCommand, model, commandHistory,
+                Messages.MESSAGE_INVALID_MODULE);
 
-        model.addModule(new ModuleBuilder(DISCRETE_MATH).withYear(5).build());
+        model.addModule(new ModuleBuilder(DISCRETE_MATH).withYear(TypicalModules.YEAR_TWO).build());
```

Comment:
At some point I think we should agree to a standard list of modules for TypicalModule and anyone who needs something more should add instead of changing. If we can't come to an agreement on the standard list of modules then we have to create them on every single test class which is really unintuitive. I don't see why a person would have to change it though.

Reply:
I guess its just the fact that the code doesn't feel very intuitive/informative to me, as in, doing this doesn't tell me that the newly added module is gonna have a different year as compared to the original one, it only tells me that it will have year=2. But otherwise, its a trivial issue.